guides/doc-Release_Notes/topics/foreman-3.10.0.adoc

```diff
@@ -0,0 +1,130 @@
+= Foreman 3.10.0
+
+A full list of changes is available on https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1789[Redmine]
+
+== Headline Features
+
+=== Preparing for Ruby 3
+
+This release includes some features that are preparing the project to run on Ruby 3. As this is a large endeavour, it will take multiple releases and features to complete. See some of the features in the release notes for more details.
```

Comment:
In the docs we have one sentence per line.
```suggestion
This release includes some features that are preparing the project to run on Ruby 3.
As this is a large endeavour, it will take multiple releases and features to complete.
See some of the features in the release notes for more details.
```